scripts/index.js

```diff
@@ -0,0 +1,105 @@
+    if (tempNum >= 80) {
+        landscapeView.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if (tempNum >= 70) {
+        landscapeView.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (tempNum >= 60) {
+        landscapeView.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (tempNum >= 50) {
+        landscapeView.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else{
+        landscapeView.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
+}
+const updateTemp = temp => {
+    const tempVal = document.getElementById("temp");
+    tempVal.textContent = temp;
+    updateTempColor(temp);
+    updateLandscape(temp);
+}
+
+const onIncreaseClick = () => {
+    temp += 1;
+    document.getElementById("temp").innerHTML = temp;
+    updateTemp(temp);
+};
+const onDecreaseClick = () => {
+    temp -= 1;
+    document.getElementById("temp").innerHTML = temp;
+    updateTemp(temp);
+};
+
+//let skyChoice = $("#skyChoice option:selected").val();
+selectElement.addEventListener('change', (event) => {
+    const updateSky = () => {
+        // const skyChoice = document.querySelector("skyChoice").value;
+        // const skyVal = document.getElementById("sky");
+
+    //     if (skyChoice === "Sunny") {
+    //         skyVal.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";   
+    //     } else if (skyChoice === "Cloudy") {
+    //         skyVal.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    //     } else if (skyChoice === "Rainy") {
+    //         skyVal.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";    
+    //     } else if (skyChoice === "Snowy") {
+    //         skyVal.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    //     }
+        const skyChoice = document.querySelector(`${event.target.value}`);
+        const skyVal = document.getElementById("sky");
+        switch (skyChoice) {
+            case "Sunny":
+                skyVal.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+                break;
+            case "Cloudy":
+                skyVal.textContent = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+                break;
+            case "Rainy":
+                skyVal.textContent = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+                break;
+            case "Snowy":
+                skyVal.textContent = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+                break;
+            default:
+                skyVal.textContent = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+        }
+    }
+});
+
+const updateCity = () => {
+    const newCity = document.getElementById("city").value;
+    const updatedCity = document.getElementById("header-city");
+    updatedCity.textContent = newCity;
+}
+
+const registerEventHandlers = () => {
+    updateTemp(temp);
+    const onIncreaseClick = document.getElementById("increase");
+    onIncreaseClick.addEventListener("click", increaseTemp);
+    const onDecreaseClick = document.getElementById("decrease");
+    onDecreaseClick.addEventListener("click", decreaseTemp);
+    updateSky();
+    const skySelect = document.querySelector("sky");
+    skySelect.addEventListener("change", updateSky);
+    updateCity();
```

Comment:
Make sure to add the eventListener for updateCity. It should happen when a user clicks the update button.

Reply:
Add another eventListener for the reset button to reset the city name